### PR TITLE
Clean documentation 2.0

### PR DESCRIPTION
**Pull request by danielgraziano**

### CODE REVIEW

File: docs/clean/user_guide/writing_a_clean/image_management.rst [hunk 1]
```diff
@@ -0,0 +1,309 @@
+.. _clean_doc_image_management:
+
+Image Management
```
Comment (jeaubin) [inline docs/clean/user_guide/writing_a_clean/image_management.rst:3]: should you talk about the argument to pass it?   

where are those described?

File: docs/clean/user_guide/writing_a_clean/device_recovery.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+.. topic:: This may depend on:
+
+    * The result of the last executed stage.
+    * The connectivity to the device.
+    * If the last stage ran was the `connect` stage.
+
+**The following diagram explains, in detail, the flow of Device Recovery.**
+
+.. image:: ../../assets/pyATS_clean_device_recovery_diagram.png
```
Comment (jeaubin) [inline docs/clean/user_guide/writing_a_clean/device_recovery.rst:38]: text is blury, can you get a larger picture

File: docs/clean/user_guide/writing_a_clean/device_recovery.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+              tftp_boot (only if 'golden_image' from above is not defined):
+                image (list): Image to boot with.
+                ip_address (list): Management ip address to configure to reach to
+                  the TFTP server.
+                subnet_mask (str): Management subnet mask.
+                gateway (str): Management gateway.
+                tftp_server (str): Tftp server is reachable with management interface.
+
+        Clean YAML - Device Recovery Example
```
Comment (jeaubin) [inline docs/clean/user_guide/writing_a_clean/device_recovery.rst:151]: have this example as a second box, so it isnt missed.  and say on top Example

File: docs/clean/user_guide/executing_a_clean.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
+.. _clean_doc_usage_and_args:
+Executing a Clean
+=================
+
+This page contains the basic arguments required to use pyATS Clean. There are two different methods of using pyATS Clean
```
Comment (jeaubin) [inline docs/clean/user_guide/executing_a_clean.rst:5]: weird sentence This page contains the basic arguments required to use pyATS Clean. 

This page explains how to execute pyATS Clean and its arguments. ?

**Comment by jeaubin** [comment]
I got an user asking me today if health, clean works with non genie script. Can you add to the doc in the intro that this works for ANY pyats script

File: docs/clean/user_guide/writing_a_clean/device_recovery.rst (hunk 2 above)
Comment (danielgraziano) [reply docs/clean/user_guide/writing_a_clean/device_recovery.rst:38]: switched to svg image

**Pull request merged**